scripts/validate_erlang_config.erl

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env escript
+main([ConfigFile]) ->
+    {ok, Terms} = file:consult(ConfigFile),
+    io:format("~p~n",[Terms]).
```

Comment:
Yes, this was just for manually validating that the config file generated by the function I wrote was correct and able to be interpreted by Erlang.